Lib/fontTools/feaLib/testdata/spec9a.ttx

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ttFont>
+
+  <BASE>
+    <Version value="1.0"/>
+    <HorizAxis>
+      <BaseTagList>
+        <!-- BaseTagCount=None -->
```

Comment:
I omitted the count since it will be calculated anyway when compiling the table.